charts/k8s-service/values.yaml

```diff
@@ -333,3 +333,26 @@ tolerations: []
 # imagePullSecrets lists the Secret resources that should be used for accessing private registries. Each item in the
 # list is a string that corresponds to the Secret name.
 imagePullSecrets: []
+
+# google specifies Google (GKE) specific configuration to be set via arguments/env. variables
```

Comment:
I'm using the `google.` -prefix to namespace GKE-specific config values. This is something that I've seen used in other charts, as well.

Reply:
Makes sense to me! Can you make this a block comment (the ones wrapped in `#-----...`) so that it is more obvious, and we don't append other generic parameters below this?